offers/admin.py

```diff
@@ -1,3 +1,39 @@
 from django.contrib import admin
+from .models import Offer, Price, EntriesCrs
 
-# Register your models here.
+
+class MultiDBModelAdmin(admin.ModelAdmin):
+    '''
+    A ModelAdmin special configuration to tell Admin to use alternative database 
+    in offers app context.
+    '''
+    # A handy constant for the name of the alternate database.
+    using = 'offers'
```

Comment:
Ciekawe, dobre rozwiazanie. Te class methods to wszystkie ktore wymagaja sprecyzowania z jakiej DB pochodzi model?

Reply:
Tak, to jest według dokumentacji kompletna definicja class methods.
Lista wymaganych metod jest krótsza, jeśli definicja klasy dziedziczy po InlineModelAdmin - ale nie zagłębiałem się w różnice z ModelAdmin wykorzystywanym tutaj.